# HG changeset patch
# User Kai T. Ohlhus <k.ohlhus@gmail.com>
# Date 1477915145 -3600
#      Mon Oct 31 12:59:05 2016 +0100
# Branch stable
# Node ID 8e71fd674bcf59f0f2c117108427aeb18f8939ee
# Parent  a9b4476a734ec1d760b674935d738f1aa51705f6
maint: libinterp/corefcn/fft.cc: minor style fixes for C++ code.

diff --git a/libinterp/corefcn/fft.cc b/libinterp/corefcn/fft.cc
--- a/libinterp/corefcn/fft.cc
+++ b/libinterp/corefcn/fft.cc
@@ -110,17 +110,17 @@ do_fft (const octave_value_list &args, c
         return octave_value (NDArray (dims));
     }
 
   if (n_points == 1)
     {
       octave_value_list idx (ndims);
       for (octave_idx_type i = 0; i < ndims; i++)
         idx(i) = idx_vector::colon;
-      idx(dim) = idx_vector (static_cast <octave_idx_type> (0));
+      idx(dim) = idx_vector (static_cast<octave_idx_type> (0));
 
       return arg.do_index_op (idx);
     }
 
   if (arg.is_single_type ())
     {
       if (arg.is_real_type ())
         {
